ci-docker-base/Dockerfile.cilint-clang-tidy

```diff
@@ -16,11 +16,11 @@ RUN apt-get update && \
 # Build clang-tidy, copy out the binary, and remove the llvm checkout
 RUN cd ./clang-tidy-checks && \
     ./setup.sh && \
-    cp llvm-project/build/bin/clang-tidy . && \
+    cp -r llvm-project/build . && \
     rm -rf llvm-project
 
 # Link clang-tidy to custom build
-RUN ln -s $(pwd)/clang-tidy-checks/clang-tidy /bin/clang-tidy
+RUN ln -s $(pwd)/clang-tidy-checks/build/bin/clang-tidy /usr/bin/clang-tidy
```

Comment:
Since the path is relative to the binary, when it’s linked to /usr/bin is it still able to find the headers?

Reply:
Yup, it is picking it up correctly

Tested by building clang and running the following commands
```
$ ln -svf $(pwd)/build/bin/clang /usr/bin/clang
$ echo '#include<stddef.h>` | clang -xc -v -
```
Output:
```
 "/clang-tidy-checks/build/bin/clang-11"
...
-resource-dir /clang-tidy-checks/build/lib/clang/11.0.0 
...
-internal-isystem /clang-tidy-checks/build/lib/clang/11.0.0/include
...
```